### ISSUE TITLE
Sort GlassBR Terminology and Definitions

### ISSUE THREAD

**Issue by samm82** [JacquesCarette/Drasil#1245]
(Continued from #835 and to be completed after #1244)

The Terminology and Definitions section in GlassBR should be sorted alphabetically (automatically by Drasil).

**Comment by JacquesCarette** [comment]
If something ought to be sorted alphabetically, then Drasil should be set up to do it, a human should not ever have to 'sort'.

**Comment by samm82** [comment]
That's the intention of this issue - sorry if I wasn't clear.

**Comment by samm82** [comment]
Duplicate of #1432
